exampleRunScripts/run_simPressShaped_coFilt

```diff
@@ -0,0 +1,130 @@
+% Bfield            = Magnetic field strength in [T]
+% Npts              = number of spectral points
+% sw                = spectral width [Hz]
+% Bfield            = magnetic field strength [Tesla]
+% lw                = linewidth of the output spectrum [Hz]
+% thkX              = slice thickness of x refocusing pulse [cm]
+% thkY              = slice thickness of y refocusing pulse [cm]
+% fovX              = full simulation FOV in the x direction [cm]
+% fovY              = full simulation FOV in the y direction [cm]
+% nX                = number of spatial grid points to simulate in x-direction
+% nY                = number of spatial grid points to simulate in y-direction
+% taus              = vector of pulse sequence timings  [ms]
+% spinSys           = spin system to simulate 
+%
+% OUTPUTS:
+% out_posxy         = Simulation results, spatially resolved.
+% out               = Simulation results, summed over all space.
+
+% ************INPUT PARAMETERS**********************************
+refocWaveform='sampleRefocPulse.pta'; %name of refocusing pulse waveform.
+refTp=5; %duration of refocusing pulses[ms]
+Npts=2048; %number of spectral points
+sw=2000; %spectral width [Hz]
+Bfield=3; %magnetic field strength [Tesla]
+lw=2; %linewidth of the output spectrum [Hz]
+thkX=1.66; %slice thickness of x refocusing pulse [cm]
+thkY=1.66; %slice thickness of y refocusing pulse [cm]
+fovX=4; %size of the full simulation Field of View in the x-direction [cm]
+fovY=4; %size of the full simulation Field of View in the y-direction [cm]
+nX=16; %Number of grid points to simulate in the x-direction
+nY=16; %Number of grid points to simulate in the y-direction
+tau1=6; %TE1 for first spin echo [ms]
+tau2=6; %TE2 for second spin echo [ms]
+spinSys='Lac'; %spin system to simulate
+% ************END OF INPUT PARAMETERS**********************************
+
+%set up spatial grid
+x=linspace(-fovX/2,fovX/2,nX); %X positions to simulate [cm]
+y=linspace(-fovY/2,fovY/2,nY); %y positions to simulate [cm]
+
+%Load RF waveform
+refRF=io_loadRFwaveform(refocWaveform,'ref',0);
+
+
+gamma=42577000; %gyromagnetic ratio
+
+%Load spin systems
+load spinSystems
+sys=eval(['sys' spinSys]);
+
+%Resample refocusing RF pulse from 400 pts to 100 pts to reduce
+%computational workload
+refRF=rf_resample(refRF,100);
+
+Gx=(refRF.tbw/(refTp/1000))/(gamma*thkX/10000); %[G/cm]
+Gy=(refRF.tbw/(refTp/1000))/(gamma*thkY/10000); %[G/cm]
+
+%Initialize structures:
+out_posxy=cell(length(x),length(y));
+out=struct([]);
+
+%loop through space: Don't forget to initialize the parallel processing
+%toolbox workers using 'matlabpool open N' (for N workers, 12 max).
+
+%for X=1:length(x);
+parfor X=1:length(x)
+    for Y=1:length(y)
+                disp(['Executing X-position ' num2str(X) ' of ' num2str(length(x)) ', '...
+                    'Y-position ' num2str(Y) ' of ' num2str(length(y))])
+                out_posxy{X}{Y}=sim_press_shaped_coFilt(Npts,sw,Bfield,lw,sys,tau1,tau2,...
```

Comment:
If you change the name of 'sim_press_shaped_coFilt.m' to 'sim_press_shaped.m', then you can change this line accordingly.